src/event_loop.rs

```diff
@@ -93,11 +94,14 @@ impl<T> EventLoopBuilder<T> {
     ///
     /// - **iOS:** Can only be called on the main thread.
     #[inline]
-    pub fn build(&mut self) -> EventLoop<T> {
-        EventLoop {
-            event_loop: platform_impl::EventLoop::new(&self.platform_specific),
+    pub fn build(&mut self) -> Result<EventLoop<T>, CreationError> {
+        Ok(EventLoop {
+            event_loop: match platform_impl::EventLoop::new(&self.platform_specific) {
+                Ok(event_loop) => event_loop,
+                Err(err) => return Err(CreationError::EventLoop(format!("Error creating event loop: {}", err))),
```

Comment:
I think you're unnecessarily converting the `CreationError::InitializeBackend` error into an `CreationError::EventLoop` here

Reply:
Fixed